pimcore/lib/Pimcore/Bundle/AdminBundle/Controller/Admin/ElementController.php

```diff
@@ -463,6 +463,7 @@ public function typePathAction(Request $request)
             $data['index'] = $element->getIndex();
         } else {
             $element = DataObject::getById($id);
+            $data['index'] = $element->getIndex();
```

Comment:
index or idx ?

Reply:
It's used [here](https://github.com/DivanteLtd/pimcore/blob/f1ecfa4f688540c2fd4f2648794fcf432f461f6d/web/pimcore/static6/js/pimcore/treenodelocator.js#L126). "Index" key is in the same convention like for documents.